java/arcs/core/data/Plan.kt

```diff
@@ -20,9 +20,10 @@ import arcs.core.util.lens
  * A [Plan] is usually produced by running the build time Particle Accelerator tool, it consists
  * of a set of specs for handles, particles used in a recipe, and mappings between them.
  */
-open class Plan(
+data class Plan(
```

Comment:
I'm glad this is becoming a data class.

Reply:
Keep in mind that equals()/hashCode() will still be wonky because Schema objects have lambda's on them, and those lambda's are being compared by reference equality. This'll be fixed when they're replaced with the expression DSL.